pipeline_kinesis--0.9.1.sql

```diff
@@ -0,0 +1,72 @@
+-- complain if script is sourced in psql, rather than via CREATE EXTENSION
+\echo Use "CREATE EXTENSION pipeline_kinesis" to load this file. \quit
+
+CREATE TABLE pipeline_kinesis_endpoints (
```

Comment:
Create everything under the `pipeline_kinesis` schema. See https://github.com/pipelinedb/pipeline_kafka/commit/48a367cfd876ad591c7befa93322310de4dfadb8